src/index.js

```diff
@@ -6,10 +6,10 @@ const bodyParser = require('body-parser');
 const jwt = require('express-jwt');
 const url = require('url');
 const requireAll = require('require-all');
-const context = require('./context');
-const initContext = require('./context/init');
+const { init, inject } = require('@forestadmin/context');
+const serviceBuilder = require('./context/service-builder');
 
-context.init(initContext);
+init(serviceBuilder);
```

Comment:
[info]
init/inject method should disappear after all inject calls are removed from files.